.github/workflows/bump-version-flake.yml

```diff
@@ -0,0 +1,48 @@
+name: Bump nixpacks-flake version
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+env:
+  AWK_CMD: '{sub(/${{ env.REGEX }}/, "v0.0.21"); print}'
```

Comment:
Should `v0.0.21` be hardcoded here?

Reply:
nop! I was testing some different regex and forgot to replace hard coded values...